cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -597,6 +598,14 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                     this.$("#id_not_timed").prop('checked', true).trigger('change');
                 };
 
+                selectGeneralSettings = function() {
+                   this.$(".modal-section a#general_settings").click();
```

Comment:
We don't specify the element names in our JavaScript, so I'd change all such references in this PR to be of the form ``".modal-section #general_settings"``.